frontend/src/components/elements/Table/Table.tsx

```diff
@@ -16,131 +16,95 @@
  */
 
 import React, { ReactElement } from "react"
-import { Table as ReactTable } from "reactstrap"
 import { toFormattedString } from "lib/format"
 import { Map as ImmutableMap } from "immutable"
 import { dataFrameGet, dataFrameGetDimensions } from "lib/dataFrameProto"
 import withFullScreenWrapper from "hocs/withFullScreenWrapper"
 import "./Table.scss"
+import { range } from "lodash"
+
+type DataFrame = ImmutableMap<string, any>
 
 /**
  * Functional element representing a DataFrame.
  */
 export interface TableProps {
-  element: ImmutableMap<string, any>
+  element: DataFrame
+}
+
+function generateTableCell(
+  df: DataFrame,
+  rowIdx: number,
+  colIdx: number
+): ReactElement {
+  const { contents, styles, type } = dataFrameGet(df, colIdx, rowIdx)
+  const formattedContents = toFormattedString(contents)
+
+  switch (type) {
+    case "corner":
+      return <td key={colIdx}>&nbsp;</td>
```

Comment:
This took me awhile to figure out. I assume this is the top left corner that we keep empty?

Reply:
Yes